src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -114,6 +119,21 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
             {
                 telemetry.TelemetryEvent.AddPiiData(ProjectFilePath, _request.Project.FilePath);
 
+                bool areNamespacesEnabled = _request?.PackageNameSpaces.AreNamespacesEnabled ?? false;
```

Comment:
`vs/nuget/projectrestoreinformation` event for PR projects.

Reply:
@nkolev92 
It looks calculation for `NewPackagesInstalledCount` is not correct, it always return 1 even though 3 packages are restored.
Maybe I'll make follow up PR this event for correct calculation. Is it ok?
https://github.com/NuGet/NuGet.Client/blob/048c1f435952666c470d85724b3d4ea1e100c168/src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs#L386

Reply:
I'm not sure I follow why this comment is relevant.